internal/api/device.go

```diff
@@ -153,7 +153,8 @@ type DevicesResponse struct {
 
 //DeviceACLTemplateRequest describes request for updating device ACL template
 type DeviceACLTemplateRequest struct {
-	TemplateUUID *string `json:"aclTemplateUuid"`
+	TemplateUUID        *string `json:"aclTemplateUuid"`
```

Comment:
I think we'll want `omitempty` for both fields to prevent sending literal `null` on each request.

Reply:
Fixed.